lib/l10n/app_es.arb

```
@@ -351,17 +351,32 @@
   "calculating_text": "Calculando...",
```

RuboCop (1.86.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)